lib/breaker.js

```diff
@@ -92,9 +92,12 @@ Breaker.prototype._run = function _run(/*args...n, callback*/) {
         var error = new Error(self.settings.timeoutErrMsg || 'Command timeout.');
         error.name = 'commandTimeout';
         error.code = 'ETIMEDOUT';
+        if (context){
+          error.context = context;
+        }
         timer = undefined;
         self._pendingClose = false;
-        self.emit('timeout');
+        self.emit('timeout', error);
```

Comment:
any advantage of emitting the error along with 'timeout'? Since we are already passing it back to the callback